packages/plugin-workspace-tools/sources/commands/focus.ts

```diff
@@ -86,6 +86,7 @@ export default class WorkspacesFocus extends BaseCommand {
           workspace.manifest.devDependencies.clear();
         }
       } else {
+        workspace.manifest.setRawField(`_excluded`, true);
```

Comment:
I think the field name could be a bit more descriptive.

Reply:
Renamed into `_evictedFromFocusedInstall`